CHANGELOG.md

```diff
@@ -1,3 +1,292 @@
+  ([#1080](https://github.com/kinvolk/lokomotive/pull/1080)).
+- pkg/terraform: improvements
+  ([#1027](https://github.com/kinvolk/lokomotive/pull/1027)).
+- cli/cmd: cleanups part 1
+  ([#1013](https://github.com/kinvolk/lokomotive/pull/1013)).
+- test/components/kubernetes: remove kubelet pod when testing node labels
+  ([#1052](https://github.com/kinvolk/lokomotive/pull/1052)).
+- Remove usage of template_file
+  ([#1046](https://github.com/kinvolk/lokomotive/pull/1046)).
+- test/*: de-duplicate value timeout and retryInterval
+  ([#1049](https://github.com/kinvolk/lokomotive/pull/1049)).
+- Packet: Read BGP peer address from metadata service
+  ([#1010](https://github.com/kinvolk/lokomotive/pull/1010)).
+- pkg/assets: cleanup exported API
+  ([#936](https://github.com/kinvolk/lokomotive/pull/936)).
+- Cobra updated to v1.1.1
+  ([#1082](https://github.com/kinvolk/lokomotive/pull/1082)),
+  ([#1091](https://github.com/kinvolk/lokomotive/pull/1091)).
+- cli/cmd: cleanups part 2
+  ([#1015](https://github.com/kinvolk/lokomotive/pull/1015)).
+- Add github actions
+  ([#1074](https://github.com/kinvolk/lokomotive/pull/1074)).
+- Makefile: use latest Go when building in Docker
+  ([#1083](https://github.com/kinvolk/lokomotive/pull/1083)).
+- cli/cmd: cleanups part 3
+  ([#1018](https://github.com/kinvolk/lokomotive/pull/1018)).
+- Add new CI config for Packet based FLUO testing
+  ([#1110](https://github.com/kinvolk/lokomotive/pull/1110)).
+
+### Updating from v0.4.1
+
+#### Configuration syntax changes
+
+There have been some minor changes to the configurations of worker nodes. The data type of `labels` and `taints` has been changed from `string` to `map(string)` for the AWS and Packet platforms.
+
+##### Old:
+
+```sh
+labels = "testing=true"
+
+taints = "nodeType=storage:NoSchedule"
+```
+
+##### New:
+
+```sh
+labels = {
+  "testing" = "true"
+}
+
+taints = {
+  "nodeType" = "storage:NoSchedule"
+}
+```
+
+This release also changes the default `cluster.oidc.client_id` value from `gangway` to `clusterauth`. This setting must match `gangway.client_id` and `dex.static_client.id`. If you use default settings for oidc you'll need to add `client_id = "gangway"` or change the `static_client.id` and `client_id` parameters for dex and gangway to `clusterauth` respectively.
+
+##### Old:
+
+```sh
+packet {
+  oidc {
+    client_id = "gangway"
+  }
+}
+```
+
+##### New:
+
+```sh
+packet {
+  oidc {
+    client_id = "clusterauth"
+  }
+}
+```
+
+#### Cluster update steps
+
+Due to [Terraform](https://github.com/kinvolk/lokomotive/pull/824) and [Kubernetes](https://github.com/kinvolk/lokomotive/pull/1030) updates to v0.13+ and v1.19.3 respectively, some manual steps need to be performed when updating. In your cluster configuration directory, follow these steps:
+
+1. Update local Terraform binary to version v0.13.X. You can follow [this guide](https://learn.hashicorp.com/tutorials/terraform/install-cli.) to do that.
```

Comment:
Before upgrading anything, I would say to ensure, that cluster is right now in a healthy state by running `lokoctl cluster apply` using `v0.4.1` version. Also, as we do not support version-jumping, I'd mention, that if your cluster is older, then upgrade to `v0.4.1` and only then proceed with upgrades to `v0.5.0`.

This will also make sure that the right `kubeconfig` file is in assets directory before the upgrades.